service.onmute/settings.py

```diff
@@ -0,0 +1,24 @@
+#   Copyright (C) 2017 Lunatixz
+#
+#
+# This file is part of OnMute
+#
+# OnMute is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OnMute is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with OnMute.  If not, see <http://www.gnu.org/licenses/>.
+
+import xbmcaddon
```

Comment:
I can change it... but when you really break it down it's a style preference.  If I had a function` f(x=0)` and I wrote `x=0 f(x)` chances are you'd ignore it, rather than suggest I change it to `f(x)`. However In this case it's part of kodis api so it's drawing attention. 

If it's a real deal issue I'll gladly change it, let me know :)